src/main/java/edu/oakland/production/database/DatabaseGisInterfaceImplementation.java

```diff
@@ -0,0 +1,35 @@
+package edu.oakland.production.database;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.production.database.DatabaseGisManagerStub;
+
+public class DatabaseGisInterfaceImplementation {
+
+  DatabaseGisManager manager;
```

Comment:
Add an extra line between the private attribute and the constructor.
```suggestion
  DatabaseGisManager manager;
  
```